integration/testutil.go

```diff
@@ -703,7 +726,66 @@ func setup() ([]func() error, error) {
 		compose.WithStdio(testutil.TestingLogDest()),
 	}
 
-	return compose.Build(composeYaml, cOpts...)
+	ctx := namespaces.WithNamespace(context.Background(), "soci-testing")
+	client, err := containerd.New(containerdAddress)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to containerd: %w", err)
+	}
+	defer client.Close()
+
+	testImages := []string{alpineImage, ubuntuImage, drupalImage, nginxImage, rabbitmqImage, rabbitmqShaImage, rethinkdbShaImage}
```

Comment:
Future additions to the image constants might not get added here. At minimum, I suggest adding a comment near https://github.com/awslabs/soci-snapshotter/pull/428/files#diff-a1077885a297cb6bcfbde19b03d7c04d94209ca38a3210644c969726aff4d985R82 pointing here. More preferably, move the creation of this slice there.